core/src/mindustry/world/blocks/ItemSelection.java

```diff
@@ -85,9 +85,11 @@ public static <T extends UnlockableContent> void buildTable(@Nullable Block bloc
 
         Table main = new Table().background(Styles.black6);
         if(rowCount > rows * 1.5f){
-            search = main.field(null, text -> rebuild.run()).width(40 * columns).padBottom(4).left().growX().get();
-            search.setMessageText("@players.search");
-            main.row();
+            main.table(s -> {
```

Comment:
what is the purpose of this?

Reply:
It just looks oddly weird because every other search bar has that zoom icon.